### PR TITLE
Support replay URLs in /join

### PR DESCRIPTION
**Pull request by AnnikaCodes**

### CODE REVIEW

File: server/chat-commands/moderation.ts [hunk 1]
```diff
@@ -412,6 +412,7 @@ export const commands: ChatCommands = {
 		if (target.startsWith('http://')) target = target.slice(7);
 		if (target.startsWith('https://')) target = target.slice(8);
 		if (target.startsWith(`${Config.routes.client}/`)) target = target.slice(Config.routes.client.length + 1);
+		if (target.startsWith(`${Config.routes.replays}/`)) target = `battle-${target.slice(Config.routes.replays.length + 1)}`;
```
Comment (mia-pi-git) [inline server/chat-commands/moderation.ts:415]: Config.routes doesn't start with https://, so you may want to check for that as well.

Comment (AnnikaCodes) [reply server/chat-commands/moderation.ts:415]:
```ts
		if (target.startsWith('http://')) target = target.slice(7);
		if (target.startsWith('https://')) target = target.slice(8);
```
It's checked for just above.

**Comment by Zarel** [comment]
Looks good, thanks!

**Comment by AnnikaCodes** [comment]
@Zarel bump

**Comment by Zarel** [comment]
Huh idk how I forgot about this.

**Pull request merged**